README.md

```diff
@@ -180,6 +180,10 @@ CSS and SASS can be added or imported into the `styles/styles.scss`. You can als
 
 Javascript can be added to the admin UI or site UI by adding or importing code into the `js/admin.js` or `js/app.js` files respectively. This template uses [esbuild](https://esbuild.github.io/) to bundle your javascript and fingerprint the files in the site build.
 
+### Managing environment variables
+
+Environment variables are set and managed on [pages.cloud.gov](https://pages.cloud.gov/)
```

Comment:
**question (non-blocking)**: Do we want to note here that `BASEURL` is already defined by the platform and that's the one that we use?

Reply:
Well, answer the upcoming slack invite